File: update_user.php
<?php
	session_start();
	include 'db_connection.php';
	$new_values = '';
	$table = $_POST['type'];
	if(isset($_POST['FirstName'])){
		$new_values.="FirstName='".$_POST['FirstName']."',";
	}
	if(isset($_POST['LastName'])){
		$new_values.="LastName='".$_POST['LastName']."',";
	}
	if(isset($_POST['Password'])){
		$new_values.="Password='".$_POST['FirstName']."',";
	}
	$sql_query = "UPDATE ".$table." SET ".substr($new_values, 0, strlen($new_values)-1)." WHERE ".$table."ID=".$_SESSION['id'];
	if(!$result = $db->query($sql_query)){
    	echo $db->error;
  	}
  	else{
  		echo '0';
  	}
?>

File: update_user.py
#!/usr/bin/python

import cgi, cgitb, json, MySQLdb, MySQLdb.cursors, db_connection
cgitb.enable()

posted_data = cgi.FieldStorage()
values_to_change = list();
if 'FirstName' in posted_data:
	values_to_change.append('FirstName="'+posted_data['FirstName'].value+'"')
if 'LastName' in posted_data:
	values_to_change.append('LastName="'+posted_data['LastName'].value+'"')
if 'Password' in posted_data:
	values_to_change.append('Password="'+posted_data['Password'].value+'"')

values_to_change =  ",".join(values_to_change);

cursor = db_connection.get_connection()

id = posted_data['id'].value
type = posted_data['type'].value
print """content-type:text/html

"""


sql_statement = 'UPDATE '+type+' SET '+values_to_change+' WHERE '+type+'ID='+id
cursor.execute(sql_statement)
print cursor.rowcount